spec/enumerables_spec.rb

```diff
@@ -0,0 +1,148 @@
+require './lib/enumerables'
+
+RSpec.describe Enumerable do
+    let(:arr) { [1, 2, 3, 4, 3] }
+    let(:new_arr) { [ ] }
+    let(:arg) { 23 }
+    let(:count) { 5 }
+    let(:word) {['ChristLord', 'Living', 'Life', 'Love']}
+
+
+    describe '#.my_each' do
+
+        context 'When array and block are given'do
```

Comment:
I think you don't need to mention the array here. The message 'when the block is given' is enough to describe it.